main/templates/main/snippets/home.html

```diff
@@ -0,0 +1,38 @@
+<div class="container position-relative z-5 pt-0 mt-4 mt-lg-5" data-bs-theme="dark">
+    <div class="row justify-content-center text-center pt-md-2 pt-lg-0 pt-xl-2 pb-0 mb-md-2">
+        <div class="col-lg-10 col-xl-9 col-xxl-8">
+            <a class="btn btn-primary btn-sm fs-sm order-lg-3 d-none d-sm-inline-flex"
+                href="{% url 'index' %}">
+                <i class="ai-login fs-xl me-2 ms-n1"></i>
+                Tell us about yourelf...
+            </a>
+        </div>
+    </div>
+    <div class="row justify-content-center text-center pt-md-2 pt-lg-0 pt-xl-2 pb-0 mb-md-2">
+        <div class="col-lg-10 col-xl-9 col-xxl-8">
+            <a class="btn btn-primary btn-sm fs-sm order-lg-3 d-none d-sm-inline-flex"
+                href="{% url 'index' %}">
+                <i class="ai-login fs-xl me-2 ms-n1"></i>
+                ...understand your skills...
+            </a>
+        </div>
+    </div>
+    <div class="row justify-content-center text-center pt-md-2 pt-lg-0 pt-xl-2 pb-0 mb-md-2">
+        <div class="col-lg-10 col-xl-9 col-xxl-8">
+            <a class="btn btn-primary btn-sm fs-sm order-lg-3 d-none d-sm-inline-flex"
+                href="{% url 'index' %}">
+                <i class="ai-login fs-xl me-2 ms-n1"></i>
+                ...explore recommended career paths
+            </a>
+        </div>
+    </div>
+    <div class="row justify-content-center text-center pt-md-2 pt-lg-0 pt-xl-2 pb-0 mb-md-2">
+        <div class="col-lg-10 col-xl-9 col-xxl-8">
+            <a class="btn btn-primary btn-sm fs-sm order-lg-3 d-none d-sm-inline-flex"
+                href="{% url 'index' %}">
+                <i class="ai-login fs-xl me-2 ms-n1"></i>
+                Understand your team
+            </a>
+        </div>
+    </div>
+</table>
```

Comment:
I think this is a mis-matched end tag?
```suggestion
</div>
```